packages/web-app-user-management/src/search/filter/index.ts

```diff
@@ -0,0 +1,50 @@
+import { SearchProvider } from 'search/src/types'
+import { EventBus } from 'web-pkg/src/event'
+import VueRouter, { Route } from 'vue-router'
+
+function $gettext(msg) {
+  return msg
+}
+
+const kind = (currentRoute: Route): 'users' | 'groups' | undefined => {
+  switch (currentRoute.name) {
+    case 'user-management-users':
+      return 'users'
+    case 'user-management-groups':
+      return 'groups'
+    default:
+      return undefined
+  }
+}
+
+export default class Provider extends EventBus implements SearchProvider {
+  public readonly id: string
+  private readonly router: VueRouter
+
+  constructor(router: VueRouter) {
+    super()
+
+    this.id = 'usersManagement.filter'
+    this.router = router
+  }
+
+  public get label(): string {
+    return $gettext(`Search ${kind(this.router.currentRoute)} ↵`)
```

Comment:
This doesn't work in transifex. I'm afraid you have to go the repetitive way here with:
```js
switch(kind(this.router.currentRoute)) {
  case "users": return $gettext("Search users ↵")
  case "groups": return $gettext("Search groups ↵")
}
```
otherwise those strings will never land in transifex. additionally extraction from .ts files doesn't work with our current setup, so it's probably broken anyway 🙈